packages/terra-badge/src/_mixins.scss

```diff
@@ -2,3 +2,34 @@
   background-color: map-get($color-map, background-color);
   color: map-get($color-map, color);
 }
+
+$default-theme: 'consumer' !default;
+$bundled-themes: 'consumer' 'ocs' !default;
+
+@mixin terra-consumer-theme {
+  @if index($bundled-themes, 'consumer') {
```

Comment:
Wrapped the `class` and `:root` selector generated by the mixin in `:global` in this commit: https://github.com/cerner/terra-core/pull/732/commits/38794f6d48344762ac35788da9d04a3b16150798